frontend/src/components/UserDashboard/WeightProgression/WeightProgression.tsx

```
@@ -41,7 +43,12 @@ export const WeightProgression = () => {
               <CardContent className="flex flex-col  gap-2">
                 <div className="flex flex-col items-center lg:flex-row gap-2">
                   <WeightCalendar weighIns={weighIns} />
-                  <WeightChart weighIns={weighIns} />
+
+                  <GenericCarousel
+                    dotPlacement="Top"
+                    hideControls
+                    carouselItems={[<WeightChart weighIns={weighIns} />, <ProgressNoteWrapper />]}
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)